index.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+const Promise = require('bluebird');
```

Comment:
you don't need bluebird as we are in ES2016 Promise are native, we don't use bluebird